directord/drivers/__init__.py

```diff
@@ -281,6 +281,11 @@ def job_init(self):
 
         pass
 
+    def job_stop(self):
```

Comment:
do we need this? I think the message driver should use the existing job_close from the driver abstract.